pkg/collector/python/loader.go

```diff
@@ -171,17 +171,19 @@ func (cl *PythonCheckLoader) Load(config integration.Config) ([]check.Check, err
 		// Let's use the module namespace to try to decide if this was a
 		// custom check, check for py3 compatibility
 		var checkFilePath *C.char
+		var goCheckFilePath string
 
 		fileAttr := TrackedCString("__file__")
 		defer C._free(unsafe.Pointer(fileAttr))
 		// get_attr_string allocation tracked by memory tracker
 		if res := C.get_attr_string(rtloader, checkModule, fileAttr, &checkFilePath); res != 0 {
-			reportPy3Warnings(name, C.GoString(checkFilePath))
+			goCheckFilePath = C.GoString(checkFilePath)
 			C.rtloader_free(rtloader, unsafe.Pointer(checkFilePath))
 		} else {
-			reportPy3Warnings(name, "")
 			log.Debugf("Could not query the __file__ attribute for check %s: %s", name, getRtLoaderError())
```

Comment:
Should we use a `warn` level here ? or that's something expected to happen in some cases ?

```suggestion
			log.Warnf("Could not query the __file__ attribute for check %s: %s", name, getRtLoaderError())
```

Reply:
@AlexandreYang  Since the `__file__` attribute is optional, perhaps a warning is too much? Maybe `Info` would make more sense as it is not required. That said, it should be available in most cases. The python 2.7 docs state:

```
__file__ is the pathname of the file from which the module was loaded, if it was loaded from a file. The __file__ attribute is not present for C modules that are statically linked into the interpreter; for extension modules loaded dynamically from a shared library, it is the pathname of the shared library file.
```

I don't really have a strong opinion. We can change it and lower it to debug in the future if it proves to be too much.